### DISCUSSION TITLE
Changing default listening port on gotenberg 7.X

### DISCUSSION THREAD

**Discussion by mdros** [gotenberg/gotenberg#570] [Q&A]
Hello,

Is it possible to change listening port on gotenberg 7.X without using `--api-port=X` flag and using environmental variables instead?  I found a way to do it with variables but it was for versions 6.X. 

Any help appreciated,
thanks!

**Comment by gulien** [comment]
Hello @mdros,

From the [documentation](https://gotenberg.dev/docs/modules/api#properties):

```
--api-port-from-env string  Set the environment variable with the port on which the API should listen - override the default port
```
